common/persistence/sql/sqlplugin/postgres/db.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package postgres
+
+import (
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+
+	"github.com/uber/cadence/common/persistence/sql/sqlplugin"
+)
+
+// db represents a logical connection to mysql database
+type db struct {
+	db        *sqlx.DB
+	tx        *sqlx.Tx
+	conn      sqlplugin.Conn
+	converter DataConverter
+}
+
+var _ sqlplugin.DB = (*db)(nil)
+var _ sqlplugin.Tx = (*db)(nil)
+
+func (pdb *db) GetDriverName() string {
```

Comment:
where is this used ?

Reply:
It's used in "func (m *sqlStore) GetName() string {" in sql/common.go".

Reply:
You are right. The one that is being used is GetPluginName(). This one is duplicated when I rename from Driver to Plugin.